### PR TITLE
added for you page and template journals on the centralized feed

### PR DESCRIPTION
**Pull request by Justin200104**

### CODE REVIEW

**Review by colefoster — APPROVED** [review]
Looks good, prettier tests are failing though. It may be easier if you have the prettier vscode plugin. But other than that all good

File: components/ForYouPage.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      <View style={styles.slide}>
+        <ForYouCard/>
+      </View>
+      <View style={styles.slide}>
+        <ForYouCard/>
+      </View>
+      <View style={styles.slide}>
+        <ForYouCard/>
+      </View>
+      {/* Add more cards as needed */}
```
Comment (ehharvey) [inline components/ForYouPage.tsx:19]: This section should be a loop I think. Can you create a constructor for `<ForYouCard>`?

File: components/ForYouCard.tsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  title: {
+    fontSize: 24,
+    fontWeight: 'bold',
+    marginBottom: 10,
+  },
+  description: {
+    fontSize: 16,
+    textAlign: 'center',
+  },
+});
```
Comment (ehharvey) [inline components/ForYouCard.tsx:115]: I feel like styling should be in a separate area, but I'm not 100% sure. @colefoster @Justin200104 thoughts?

**Review by ehharvey — DISMISSED** [review]
Lastly, we had talked about putting the feed on the debug screen, but IDK tbh. Cole showed me `storybook` which is a web app thingy to look at components. Not sure if that would be better than a debug screen. Thoughts?

File: components/ForYouCard.tsx [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    description: "Exploring the remnants of ancient ruins, I felt the weight of history upon my shoulders. Stone pillars stood tall against the backdrop of a setting sun, their weathered surfaces bearing the marks of time. As I walked among the ruins, I could almost hear the whispers of the past, tales of civilizations long gone. Nature had begun to reclaim its territory, with vines weaving through cracks in the stone. It was a humbling experience, standing amidst the remnants of a bygone era, feeling a deep connection to those who walked these paths centuries before.",
+  },
+  {
+    title: "A Serenade of Birdsong",
+    description: "Waking to a symphony of bird songs, I knew it would be a day filled with nature's melodies. The air was alive with chirps, trills, and calls, each bird adding its unique note to the chorus. I followed the sound through the forest, feeling as though I was part of this avian orchestra. Perched on a moss-covered rock, I closed my eyes and let the music wash over me. It was a reminder of the beauty and harmony of the natural world, a serenade that filled my heart with joy.",
+  },
+  // Add more journal stories as needed
+];
```
[the remark this answers is not in the document]
Comment (colefoster) [reply components/ForYouCard.tsx]: Moved text to /constants/ directory: FakeForYouCardText.ts

File: components/ForYouPage.tsx (hunk 1 above)
Comment (colefoster) [reply components/ForYouPage.tsx]: Fixed

File: components/ForYouCard.tsx (hunk 2 above)
Comment (colefoster) [reply components/ForYouCard.tsx]: I think globalized styling and component design are both good ideas, will have to be sprint 3 (or 2.5) though unfortunately

**Pull request merged**